hil/test_common.py

```diff
@@ -437,7 +438,21 @@ def obmd_cfg():
         be used as a fixture, but not declared here as such; individual
         modules should declare it as a fixture.
         """
-        portno = 8833
+        # Make the port number a function of which worker we're running on,
+        # so parallel tests don't try to use the same port. We derive the
+        # port number by stripping out all non-digit characters from the
+        # worker id, and then adding the result to a base. If the worker id
+        # isn't set, we're probably not running in parallel, so just use
+        # the base.
+        portno = 8800
+        worker = os.getenv('PYTEST_XDIST_WORKER')
+        if worker is not None:
+            portno += + int(''.join([
```

Comment:
is that stray `+` intentional? (it doesn't break anything, but does it have some purpose that I'm not seeing?)

Reply:
No, just a typo -- I've removed it.